platforms/nuttx/src/bootloader/microchip/mpfs/CMakeLists.txt

```diff
@@ -36,6 +36,15 @@ px4_add_library(arch_bootloader
 	systick.c
 )
 
+execute_process(
+	COMMAND  git show --summary --pretty="%h %d, %as"
```

Comment:
With this output, you wouldn't see if the bootloader is someone's hacked/dev version from own PC; Also, the commit's timestamp is redundant; time of build execution would be more useful, but not mandatory.

I'd like to see also git status info, whether it is dirty or not (like "git describe --dirty").

Maybe compose the info just from "git describe --dirty" and the build time?

Reply:
I am not sure if you can just pass this as one command, but something like this:
echo `git describe --dirty` `date +'%Y-%m-%d %H:%M:%S'` ?

The version then matches what PX4 is outputting, together with git commit, and the date is current build date.

Reply:
According to Matias Soini, information about the latest commit with hash and dirty is enough. The timestamp is unnecessary. Changed accordingly.